testing/lzbench/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Oleg Titov <oleg.titov@gmail.com>
+# Maintainer: Oleg Titov <oleg.titov@gmail.com>
+pkgname=lzbench
+pkgver=1.7.3
+pkgrel=0
+pkgdesc="lzbench is an in-memory benchmark of open-source LZ77/LZSS/LZMA compressors"
+url="https://github.com/inikep/lzbench"
+arch="x86_64 x86"
+license="unknown"
```

Comment:
It looks like here should be list of licenses because a lot of different sublibs commited to repo like https://github.com/inikep/lzbench/blob/master/lzf/lzf.h#L2

Reply:
I have updated APKBUILD with a list of licenses in use.